EventsExpress.Core/IServices/ITokenService.cs

```diff
@@ -0,0 +1,20 @@
+﻿using EventsExpress.Core.DTOs;
+using EventsExpress.Db.Entities;
+using System.Collections.Generic;
+using System.Security.Claims;
+using System.Threading.Tasks;
+
+namespace EventsExpress.Core.IServices
+{
+    public interface ITokenService
+    {
+        string GenerateAccessToken(UserDTO user);
+        RefreshToken GenerateRefreshToken(string ipAddress);
+        ClaimsPrincipal GetPrincipalFromExpiredToken(string token);
+        Task<AuthenticateResponseModel> RefreshToken(string token);
+        Task<bool> RevokeToken(string token);
+        void SetTokenCookie(string token);
+        string IpAddress();
```

Comment:
Consider making this method `private` if it is not being used outside of the interface.

Reply:
its method using in another authservice.

Reply:
refactored. Created private prop IPAddress